tests/Unit/Controllers/ExecutiveControllerTest.php

```
@@ -371,6 +371,76 @@ public function testProjectDashboardAttachesStoryProgress(): void
+        $project = ['id' => 5, 'name' => 'Alpha', 'updated_at' => '2025-01-01'];
+        $okrItem = ['id' => 10, 'okr_title' => 'Revenue', 'okr_description' => 'KR1: Hit targets'];
+        $spRow   = [
+            'okr_title' => 'Revenue',
+            'kr_hypothesis' => '',
+            'story_count' => 2,
+            'done_story_count' => 1,
+            'total_points' => 8,
+            'done_points' => 5,
+            'in_progress_points' => 3,
+        ];
+        $db = $this->freshExecDb([
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)